src/pages/view/DashboardsBar/styles/DashboardsBar.module.css

```diff
@@ -33,6 +37,16 @@
     height: var(--user-rows-height);
 }
 
+.offline {
+    background-color: rgb(195, 100, 100) !important;
```

Comment:
why important? and where are these non-ui colors coming from?

Reply:
Removed. This was for when the offline/online state was being mocked.